speech/speech_samples.rb

```diff
@@ -88,24 +131,30 @@ def get_async_operation_results operation_name:
   # [END get_async_operation_results]
 end
 
+require "google/cloud/speech"
+
 if __FILE__ == $PROGRAM_NAME
-  command = ARGV.shift
+  project_id = Google::Cloud::Speech.new.project_id
+  command    = ARGV.shift
 
   case command
   when "recognize"
-    transcript_from_audio_file audio_file_path: ARGV.first
+     speech_sync_recognize projec_id: project_id, audio_file_path: ARGV.first
+  when "recognize_gcs"
+    speech_sync_recognize_gcs project_id: project_id, storage_path: ARGV.first
   when "async_recognize"
-    begin_async_operation audio_file_path: ARGV.first
-  when "async_recognize_results"
-    get_async_operation_results operation_name: ARGV.first
+     speech_async_recognize projec_id: project_id, audio_file_path: ARGV.first
+  when "async_recognize_gcs"
+    speech_async_recognize_gcs project_id: project_id, storage_path: ARGV.first
   else
     puts <<-usage
 Usage: ruby speech_samples.rb <command> [arguments]
 
 Commands:
-  recognize               <audio-file.raw>
-  async_recognize         <audio-file.raw>
-  async_recognize_results <operation name>
+  recognize           audio-file.raw                Transcribe file
```

Comment:
I was thinking more like
`recognize                   [audio_file] Synchronously transcribe local file`
`recognize_gcs            [audio_file] Synchronously transcribe Cloud Storage file`
`async_recognize        [audio_file] Asynchronously transcribe local file`
`async_recognize_gcs [audio_file] Asynchronously transcribe Cloud Storage file`

Spacing is an issue though.